### PR TITLE
Quick fix: enable concurrent classification

### PR DESCRIPTION
**Pull request by gigium**
This quick-fix sets up a dynamic naming strategy for the temporary resources based on the timestamp of each execution of the classification.sh. 
This would ensure that different requests made to the API that uses pySenti4SD for classification will be consistently served by using the related working files.

### CODE REVIEW

**Comment by gigium** [comment]
@louieQ , @bateman we resolved the concurrent request problem [in the issue 7](https://github.com/collab-uniba/behaviz/issues/7) by renaming the created temp_feature files with the current timestamp. Moreover, we have changed:
- from `mkdir -p $tmpDir;` to `mkdir $tmpDir;` because the former command re-writes the whole folder, erasing the contents;
- from `rm -rf $SCRIPTDIR/temp_features` to `rm "$tmpDir/$tmpFile"` in order to delete _only_ the specific temporary file, and not the whole folder

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
+#Dockerfile used for testing
+Dockerfile
```
Comment (louieQ) [inline .gitignore]: Is this addition really necessary? Docker is not used in this repository and, generally speaking, Dockerfiles get versioned when it is.

File: classification.sh [hunk 2]
```diff
@@ -80,13 +87,14 @@ if [ ! -f $inputFile ]; then
     exit 1
 fi 
 
-mkdir -p $SCRIPTDIR/temp_features;
+mkdir $tmpDir;
```
Comment (louieQ) [inline classification.sh]: Isn't the `-p` argument necessary during the first script execution, when `$SCRIPTDIR` (the parent folder) is missing?

File: .gitignore (hunk 1 above)
Comment (gigium) [reply .gitignore]: Ok this was deleted.

File: classification.sh (hunk 2 above)
Comment (gigium) [reply classification.sh]: The `-p` argument was re-integrated as after testing, the elimination was not necessary.

**Comment by gigium** [comment]
In addition to previous commits, the `.idea` folder was removed as it should not be part of the repository.

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (louieQ) [inline .gitignore]: I'd delete this blank line to avoid tracking useless changes to `.gitignore`.

**Review by louieQ — CHANGES_REQUESTED** [review]
The IntelliJ guidelines on ["How to manage projects under Version Control Systems"](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839) state that the following IntelliJ files need to be shared:

> - All files under the `.idea` directory in the project root except the items that store user-specific settings:
>     - `workspace.xml`
>     - `usage.statistics.xml`
>     - `shelf` directory
> - All the .iml module files [...]

You deleted the whole `/.idea/` folder instead. Are you sure that this change is beneficial to the pySenti4SD project in general? If it is useful just for Behaviz, please don't merge your commits into the `master` branch, but rather into another dedicated branch.
